crates/core_arch/src/x86/sse2.rs

```diff
@@ -2653,21 +2653,17 @@ pub unsafe fn _mm_loadu_pd(mem_addr: *const f64) -> __m128d {
 #[target_feature(enable = "sse2")]
 #[cfg_attr(
     all(test, any(not(target_os = "windows"), target_arch = "x86")),
-    assert_instr(shufps, imm8 = 1)
+    cfg_attr(test, assert_instr(shufps, MASK = 2)) // FIXME shufpd expected
 )]
 #[cfg_attr(
     all(test, all(target_os = "windows", target_arch = "x86_64")),
-    assert_instr(shufpd, imm8 = 1)
+    cfg_attr(test, assert_instr(movaps, MASK = 2)) // FIXME shufpd expected
```

Comment:
Check which instruction is actually generated in the assembly output for the shuffle. Not `movaps` which has nothing to do with the shuffle.

Reply:
Try using a different `imm8` value so it generates a shuffle instruction.